network-api/networkapi/donate/migrations/0013_update_imagetextblock_with_linkblock.py

```diff
@@ -24,15 +26,31 @@ def migrate_image_text_block(source_block):
         "top_divider": source_block["value"].get("top_divider", False),
         "bottom_divider": source_block["value"].get("bottom_divider", False),
     }
+
     old_url = source_block["value"].get("url")
+
+    # Regex patterns for detecting URLs and emails
+    url_pattern = re.compile(r"^(http://|https://|www\.|[a-zA-Z0-9-]+\.[a-zA-Z0-9-]+)")
```

Comment:
Would this regex match subdomains other than `www` correctly?

Reply:
Thanks for the question @ramram-mf! This regex does. Ive tested it with domains such as:

- https://foundation.mozilla.org
- https://foundation.mozilla.org?test=test
- http://foundation.mozilla.org
- www.foundation.mozilla.org
- foundation.mozilla.org

Reply:
Great. I asked only because I did a quick review and that was the only thing that caught my attention. Didn't get as far as testing the regex at all. Thanks Daniel!